ecosystem/python/sdk/aptos_sdk/client.py

```diff
@@ -250,7 +252,7 @@ def transfer(self, sender: Account, recipient: AccountAddress, amount: int) -> s
         }
         return self.submit_transaction(sender, payload)
 
-    #:!:>bcs_transfer
+    # :!:>bcs_transfer
```

Comment:
I think `black` does this but it breaks the templating into the tutorials, have we double checked that? I _think_ that's why I wasn't using `black` on the `aptos_sdk` files in the first place.

Reply:
it worked :)